docker-compose.npm-test.yml

```diff
@@ -27,6 +27,5 @@ services:
       /bin/sh -c "
       /usr/bin/mc config host add minio-server http://minio-server:9000 USERNAME PASSWORD;
       /usr/bin/mc mb --ignore-existing minio-server/balena-pine-web-resources;
-      /usr/bin/mc anonymous set download minio-server/balena-pine-web-resources;
```

Comment:
Odly enough, this is the most important line in this commit. Now the bucket is private and files within it are only accessible if you have a signed url.